internal/storage/postgres/sensor/get.go

```diff
@@ -69,3 +71,22 @@ func (r *SensorRepository) GetSensorDataByID(ctx context.Context, id int32) ([]*
 
 	return domainData, nil
 }
+
+func (r *SensorRepository) GetCenterPoint(ctx context.Context, ids []int32) (lat, long float64, err error) {
```

Comment:
Why do we need this function?

Reply:
I thought we needed it just like in tree ?

Reply:
Have been discussed.